src/domUpdates.js

```diff
@@ -6,22 +6,22 @@ let domUpdates = {
     this.interactWithClassList('remove', `${viewLabels[2]}`, event, cardArea);
     cardArea.innerHTML = '';
     recipesList.forEach(recipe => {
-      cardArea.insertAdjacentHTML('afterbegin', `<div id='${recipe.id}'
+      cardArea.insertAdjacentHTML('afterbegin', `<div id='${recipe.id}-card'
       class='card'>
-      <header id='${recipe.id}' class='card-header'>
+      <header id='${recipe.id}-header' class='card-header'>
```

Comment:
oh yeah, I remember this from some project way back--couldn't add IDs with just numbers